scripts/o3de/o3de/engine_template.py

```diff
@@ -2331,10 +2331,15 @@ def create_repo(repo_path: pathlib.Path,
         logger.error('Remote repository URI cannot be empty.')
         return 1
     
-    # check if a repo.json file already exist in directory - can only have one repo.json file per project
+    # check if path is empty and
+    if not force and repo_path.is_dir():
+        logger.error(f'{repo_path} already exists. Use the --force command to overwrite the existing directory.')
+        return 1
```

Comment:
It's OK to write to an existing folder, that's how you would be able to write the `repo.json` file to an existing Gem folder for example.  You just don't want to overwrite any existing files.

```suggestion
```